docs/modules/ROOT/pages/index.adoc

```diff
@@ -3,3 +3,10 @@
 appcat-service-postgresql is a Commodore component to manage appcat-service-postgresql.
 
 See the xref:references/parameters.adoc[parameters] reference for further details.
+
+== Requirements
```

Comment:
I'm open for suggestions as to what to include here :)

Reply:
I added `installation.adoc` tutorial, please add your suggestions and changes there :)